File: 0136-single-number/0136-single-number.java
class Solution {
    public int singleNumber(int[] nums) {
// XOR 

//  0 0 -- 0 
//  1 1 -- 0 
//  0 1    1
//  1 0    1 
 
 int xor=0; 

 for(int i=0;i<nums.length;i++){
   xor = xor^nums[i]; 
 }
 return xor; 

}
}

File: 0136-single-number/0136-single-number.py
class Solution(object):
    def singleNumber(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        freq= Counter(nums)

        for i in freq: #i represents the key
            print(i)
            if freq[i]==1 :  #value 
                return i
        